data/remnant missions.txt

```diff
@@ -493,5 +597,764 @@ mission "Remnant: Technology Available"
+
+			label more
+			`	After examining <ship> for quite some time, he calls down to you. "I have never seen weapons designed to handle such a high heat output! Is this some kind of plasma based weapon?" he asks, gesturing at a Plasma Cannon. You nod, and he continues, "Intriguing. We have noticed that the Korath ships seem to run particularly hot, and have speculated that overheating them might be an effective means of disabling them. I would enjoy researching these a bit more. If you could acquire some for us you would be well compensated."`
+			choice
+				`	"Sure, I would be glad to work out a deal."`
+				`	"Sorry, I'm really not interested in helping Remnant research."`
+					decline
+
+			`	After a bit of haggling, he agrees to pay you one million credits in exchange for a cargo of two "Plasma Cannons."`
+				accept
+	
+	on complete
+		outfit "Plasma Cannon" -2
+		payment 1500000
+		conversation
+			`As <ship> settles onto the pad closest to the research facility, you can see the weapons specialist checking over a large cannon on a platform beneath a nearby Albatross. When he sees you starting to unload the two plasma cannons he snaps the casing closed and gestures to someone out of sight. Within moments both him and an empty flatbed are there to pick up the weapons.`
+			`	"Thank you for your help," he says. "Our Inhibitors provide us with something of a mobility advantage against the Korath, but they aren't particularly effective at disabling them. These plasma cannons will open new avenues of research for us." He pauses, then continues. "I also included a bonus for you. It is a 'finder's fee,' as we always appreciate new tech."`
+			`	The weapon tech finishes strapping down the plasma cannons, and looks up to see another Remnant approaching. He gives the new Remnant a respectful dip of his head that almost seems like a bow before turning to you and chanting, "Ah, Captain <last>, have you met Tali? She is the engineering prefect here on Viminal. She asked me to let her know when or if you managed to procure the new tech for me."`
+			`	"Well met, Newcomer among the Remnant," trills the prefect. "Fresh blood amongst us sets songs in motion that have long been in Caesura. I have a few things to finish up right now, but I hope to work with you soon." She gives you a polite nod and heads back the way she came.`
+			
+mission "Remnant: Return the Samples"
+	name "Return the Samples"
+	description "A researcher on <planet>, has asked you to return some samples to Nasqueron."
+	source Aventine
+	stopover Nasqueron
+	to offer
+		has "license: Remnant"
+		has "event: remnant: void sprite research"
```

Comment:
Unless someone uses the cheat plugin to simply give themselves the license. Keeping this restriction means that even if someone uses the cheat plugin to get the license, they will still have the mission restricted to doing them in the proper order without jumping a bunch of them.

Reply:
I can see how that would simplify things, however I think it is better this way as it is more flexible: Having the "Remnant Technology Available" as a requirement means that even if they cheat in the license and have done the Sprite missions, they can't continue it. I could see someone doing that if they couldn't get past the combat missions and wanted to see more of what happens with the Sprites, and I don't see any reason to stop them. That being said, ideally they would get the license before continuing with the sprite missions. So having both of these as conditions instead of just the one line allows the best of both worlds: It will result in most players getting the missions the intended way, while allowing players having trouble to still do more of the sprite missions if they've already done them.

Reply:
>Having the "Remnant Technology Available" as a requirement means that even if they cheat in the license and have done the Sprite missions, they can't continue it. 

I still don't see how this is an issue. Players who have trouble with the game are also likely the same players to not know how to skip a mission through save editing, and their first cheating attempt is more likely going to involve making their ship invincible or giving themselves a load of credits to buy a better ship.